CLOUD-282-doc-404 (#192)
percona/percona-xtradb-cluster-operator: source/kubernetes.rst

.. _kubernetes:

Install the |Percona Operator PXC| on Kubernetes
============================================

0. First of all, clone the percona-xtradb-cluster-operator repository:

   .. code-block:: bash

      git clone -b release-1.0.0 https://github.com/percona/percona-xtradb-cluster-operator
      cd percona-xtradb-cluster-operator

>    **Note:** *You must specify the correct branch with ``-b``

Reviewer: it is really crucial, it is the most popular bug, so the user should be careful here.

   Finally it’s time to start the operator within Kubernetes:

   .. code-block:: bash

      $ kubectl apply -f deploy/operator.yaml

4. Now that’s time to add the |Percona XtraDB Cluster| Users secrets to Kubernetes. They
   should be placed in the data section of :file:``deploy/secrets.yaml``

>    as logins and base64-encoded passwords for the user accounts

Reviewer: ```suggestion
   as base64-encoded passwords for the user accounts
```

   (see `Kubernetes
   documentation <https://kubernetes.io/docs/concepts/configuration/secret/>`__
   for details).

   **Note:** *the following command decodes the base64-encoded
   password:*
   ``$ echo -n 'plain-text-password' | base64``

>    After editing the user name and password information the user secrets are created (or

Reviewer: ```suggestion
   After editing the password information the user secrets should be created (or
```
